hyperdome/server/web.py

```diff
@@ -182,7 +185,10 @@ def counseling_complete():
             other_user = self.active_chat_user_map[sid]
             self.active_chat_user_map.pop(sid)
             self.pending_messages.pop(sid, "")
-            self.active_chat_user_map.update({other_user: ""})
+            try:
+                self.active_chat_user_map[other_user] = ""
+            except KeyError:
+                pass
```

Comment:
You might wanna explain what it means if `other_user` isn't there

Reply:
it means that the other user has already disconnected, could be more clear